src/main/java/pl/wrapper/parking/exception/DummyController.java

```diff
@@ -6,6 +6,7 @@
 import org.springframework.web.bind.annotation.RestController;
 import pl.wrapper.parking.result.Result;
 
+
```

Comment:
Think of a way to handle the someErrorType in Controller that would return the correct ResponseEntity without the need of throwing exceptions and calling ExceptionHandler.
Remember that controller returns ResponseEntity<someType> and not just someType. The return value needs to be wrapped in ResponseEntity with the correct httpStatus